components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -304,7 +304,7 @@ class SystemEngineSession(private val defaultSettings: Settings? = null) : Engin
      * See [EngineSession.exitFullScreenMode]
      */
     override fun exitFullScreenMode() {
-        // no-op
+        view?.get()?.fullScreenCallback?.onCustomViewHidden()
```

Comment:
I'm no expert on this codebase, but it looks like the only other place this method is called (in `SystemEngineView#createWebChromeClient`, override `onHideCustomView` calls `removeFullScreenView`), a few other things happen at the same time.

- `onFullScreenChange` observer is notified
- View with tag 'mosac_system_engine_fullscreen' is removed
- View with tag 'mosac_system_engine_webview' is made visible

Should we be doing any of these here as well?